corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -685,6 +685,7 @@ def javascript_libraries(context, **kwargs):
         'analytics': kwargs.pop('analytics', False),
         'hq': kwargs.pop('hq', False),
         'helpers': kwargs.pop('helpers', False),
+        'secure_cookies': settings.SECURE_COOKIES,
```

Comment:
What's this change for?

Reply:
the `core_libraries.html` file includes the `hqwebapp/js/components/pagination.js` script and that was the only spot I saw `core_libraries.html` used so I figured that's where the context would get setup for it.

Reply:
You're right that this is the context for `core_libraries.html`, but since you're not adding anything to `core_libraries.html` itself, I don't think this is doing anything. Since this is a template tag, `core_libraries.html` isn't included directly in other templates, it gets inserted by the `javascript_libraries` tag - [this](https://github.com/dimagi/commcare-hq/blob/f8f29068699aeadd0d8b0cf488acec5150926b0e/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L136) is the main place it's used.

I didn't notice earlier that `pagination.js` needs `secure_cookies`. Because of that `javascript_libraries` tag in `hqwebapp/base.html`, `pagination.js` gets included in virtually every page on HQ, so `secure_cookies` also needs to be available basically everywhere. Considering that, I now more strongly think `secure_cookies` should go into context processors instead of being passed into individual views.